### PR TITLE
create API endpoints for upsert, get, and delete for dsm-celebration feature [jira: MATD-48 (sprint 3)]

### PR DESCRIPTION
**Pull request by samim-hossain-mondal**
Features include:

- Create a celebration data (celebration_id, author(anonymously), content, createdAt(data-time))
- Update the content for a celebration data
- Read/Get all data for celebrations
- Delete a data from the celebration table

_*all endpoints are working perfectly_
_*working on swagger documentation, validations and tests_

**snapshots**
Screenshot 2023-02-16 at 3 23 52 AM
Screenshot 2023-02-16 at 3 24 52 AM
Screenshot 2023-02-16 at 3 24 18 AM
Screenshot 2023-02-16 at 3 25 04 AM
Screenshot 2023-02-16 at 3 25 38 AM

### CODE REVIEW

File: src/services/dsmSection/dsmSection.celebrationBoard.services.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+const { HttpError } = require('../../errors');
+const prisma = require('../../prismaClient');
```
Comment (sugumar-mani) [inline src/services/dsmSection/dsmSection.celebrationBoard.services.js:2]: are we missing the column `celebrationType` in the Celebration table? How are we mentiontioning it is a Celebration or an Impediment?

Alternatively, we can rename the table and files to `DailyRetro` where the type can be Celebration or Impediment. I will leave it up to you

File: src/controllers/dsmSection/dsmSection.celebrationBoard.controller.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      await celebrationBoardServices.getCelebrationById(celebrationId);
+    res.status(200).json(celebration);
+  }
+  catch (er) {
+    next(er);
+  }
+};
+
+// controller to handle POST request for creating a celebration
+const createCelebration = async (req, res, next) => {
```
Comment (sugumar-mani) [inline src/controllers/dsmSection/dsmSection.celebrationBoard.controller.js:29]: variables are well-named. Great job.

File: src/services/dsmSection/dsmSection.celebrationBoard.services.js (hunk 1 above)
Comment (samim-hossain-mondal) [reply src/services/dsmSection/dsmSection.celebrationBoard.services.js:2]: okay will discuss it with the team

File: src/controllers/dsmSection/dsmSection.celebrationBoard.controller.js (hunk 2 above)
Comment (samim-hossain-mondal) [reply src/controllers/dsmSection/dsmSection.celebrationBoard.controller.js:29]: Thank you :)

**Pull request closed without merging**